modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/package.json

```diff
@@ -0,0 +1,15 @@
+{
+	"dependencies": {
+		"frontend-editor-ckeditor-web": "*",
+		"react": "16.12.0",
+		"react-dom": "16.12.0"
+	},
+	"name": "frontend-editor-ckeditor-sample-web",
```

Comment:
This should be named `@liferay/frontend-ckeditor-sample-web` to guard against [dependency confusion attacks](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/general/security/dependency_confusion.md).

We [would like to be able to lint for this](https://github.com/liferay/liferay-frontend-projects/issues/469), but we can't do that until we've renamed all of our existing packages ([LPS-129670](https://issues.liferay.com/browse/LPS-129670) (or we implement a whitelist). cc @jbalsas, @dsanz about prioritization of that work. (My feeling: this will be soul-destroying grunt work, but I'll feel nervous until we do it... and if we don't want to do it, we should hack in the whitelist so that we can deploy a lint soon.)

Reply:
Having a white list may be read as "we want these to not have a namespace", when perhaps it should be taken as "that's the amount of pending work". Could we differentiate those in some way? And, are there chances that, once we rename all of our packages, whitelist becomes empty?

Reply:
> Could we differentiate those in some way? And, are there chances that, once we rename all of our packages, whitelist becomes empty?

Yeah, the "white list" (which I should probably call "allow list") means "it's ok for these to not be in a named scope" _and_ it will start off by coinciding with "the list of packages that are pending rename". And yes, once everything is renamed, the whitelist can be deleted. (I wouldn't actually be bothered with emptying it along the way... I'd just remove the whole thing at the end.)

Reply:
@wincent sorry about that, I really need to remember this rule, I don't want you (or anyone else) to have to rename a gazillion packages again

Reply:
The worry isn't about "gazillion" packages. Say Liferay adds "a few" packages (5, 10?) over the next few months. All we need is _one_ [bad actor](https://en.wikipedia.org/wiki/Nicolas_Cage) to come along and squat the package name on npm and we've got a potential headache. The odds of it happening are slim, and the scope of the likely damage if it does happen is not that great, but that's cold comfort because the badness of the worst case scenario is bad enough. `#bad`

![bad](https://user-images.githubusercontent.com/7074/114839998-a911a480-9dd6-11eb-9f03-71a29032a02c.gif)